Sum_Of_Squares/Sum_Of_Squares.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+    The sum of squares of first N natural numbers is given
+    by the equation ((N) * (N + 1) * (2 * N + 1)) / 6 i.e.
+    1^2 + 2^2 + 3^2 + 4^2 + 5^2.........N^2 sums upto
+    ((N) * (N + 1) * (2 * N + 1)) / 6
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+    int N;
+    cout << "Enter the value of N: " << endl;
```

Comment:
Please remove spaces around ``<<`` and ``>>`` in cout and cin statements as these are not required and the compiler simply ignores unnecessary spaces.